libs/databricks/langchain_databricks/chat_models.py

```diff
@@ -313,8 +346,20 @@ def _stream(
                 if first_chunk_role is None:
                     first_chunk_role = chunk_delta.get("role")
 
+                if stream_usage:
```

Comment:
```suggestion
                if stream_usage and (usage := chunk.get("usage")):
```


can we only include `usage` when it's present?

Reply:
makes sense!